app.js

```diff
@@ -1,11 +1,16 @@
 var express = require('express');
 var app = express();
 var project = require('./package');
+var models = require('./models');
 
 app.get('/', function (req, res) {
   res.send('Hello World!');
 });
 
+app.get('/cars', function (req, res) {
+	res.send(models.Car.findAll());
```

Comment:
will not work

Reply:
sequelize return promises, so it will not work 100%
http://docs.sequelizejs.com/manual/installation/getting-started.html
promises section